queue_job/jobrunner/runner.py

```diff
@@ -472,11 +472,10 @@ def from_environ_or_config(cls):
         return runner
 
     def get_db_names(self):
-        if config["db_name"]:
-            db_names = config["db_name"].split(",")
-        else:
-            db_names = odoo.service.db.list_dbs(True)
-        return db_names
+        db_names = config["db_name"] or []
+        if db_names:
+            return list(db_names)
```

Comment:
```suggestion
            return db_names
```

It's already a list, no?